### ISSUE TITLE
add Fig. 3

### ISSUE THREAD

**Issue by arlin** [arlin/hip_hack_howto#82]
reconstruct from data

**Comment by rvosa** [comment]
So just for my understanding because this is not clear from the different info streams: figure 3 is the figure with demographic trends through time. Are we saying that, even though there is no (?) IRB approval to share the raw data, we might show a figure?

**Comment by rvosa** [comment]
I have added a suggestion for a figure to the manuscript. This figure lumps all affirmative responses to the NEAD question about belonging to a traditionally underrepresented group (i.e. both gender and ethnicity) in one count and turns this into the percentage of participants. It would be nice if we could use this because it does show a clear trend. However, I am really unclear as to what we can and cannot put in the paper what with this IRB stuff. A number of issues now hinge on this (I've labeled them).

**Comment by rvosa** [comment]
So the decision is to NOT add a figure like this at all because any usage of NEAD data is not allowed.

**Comment by arlin** [comment]
right.  we could do gender by eyeball, but that is as far as we can go.

**Comment by hlapp** [comment]
I recognize that dropping this idea is the easiest solution, but determining gender through means other than applying for permission to release protected data, or eyeballing, is neither that hard nor is it rarely done. In fact, there are APIs and software packages for doing this based on names alone:

* A general guide: [How to identify gender at scale, ethically, and responsibly](https://civic.mit.edu/blog/natematias/best-practices-for-ethical-gender-research-at-very-large-scales)
* [Gender-API.com](https://gender-api.com/): allows 500 free requests after registration, and includes ability to upload a CSV file.
* https://genderize.io Free, but limited to 1000 requests per day (there are software interfaces to this, including in Python)
* [gender](https://cran.r-project.org/web/packages/gender/gender.pdf), an R package from rOpenSci. Integrates multiple approaches (including genderize.io).

I'm happy to try and run our names through the `gender` R package next week, or to write a shell script that uses one of the APIs directly through `curl`. Though others are welcome to beat me to the punch.

Bottom line, I think diversity in informatics is really important, and I think the data we have is interesting enough to at least try for the one demographic diversity that is relatively easy to come by even without releasing protected data.

**Comment by rvosa** [comment]
Go for it, then! With the NEAD data it looked like there was actually
something worth reporting so if we can get substantially similar results by
legal means I am all for it.

Op Sat, 10 Dec 2016 om 01:29 schreef Hilmar Lapp <notifications@github.com>

**Comment by hlapp** [comment]
I'll hold off until I get the 👍 from @arlin as in the end it will have to pass muster with his IRB, if I understand correctly.

**Comment by arlin** [comment]
Yes, thanks @hlapp, please go ahead.  I am planning a meeting with them next week to discuss the new manuscript.
